src/modals/AppSettingsModal/AppSettingsModal.js

```diff
@@ -67,17 +73,39 @@ const AppSettingsModal = () => {
           </div>
         ))}
       </div>
-      <div className={activeTab === SETTINGS_TABS.Keys ? '' : 'appsettings-modal__content'}>
-        {isElectronApp && (
-          <>
-            {activeTab === SETTINGS_TABS.Beta && <BetaTab />}
-            {activeTab === SETTINGS_TABS.General && <GeneralTab />}
-            {activeTab === SETTINGS_TABS.Keys && <ApiKeysTab />}
-          </>
+
+      <TransitionGroup
+        exit={false}
+        className={cx('content-all', {
+          'appsettings-modal__content': activeTab !== SETTINGS_TABS.Keys,
+        })}
+      >
+        {isElectronApp && activeTab === SETTINGS_TABS.Beta && (
+          <CSSTransition {...cssTransitionProps}>
+            <BetaTab />
+          </CSSTransition>
         )}
-        {activeTab === SETTINGS_TABS.Appearance && <AppearanceTab />}
-        {activeTab === SETTINGS_TABS.About && <AboutTab />}
-      </div>
+        {isElectronApp && activeTab === SETTINGS_TABS.General && (
+          <CSSTransition {...cssTransitionProps}>
+            <GeneralTab />
+          </CSSTransition>
+        )}
+        {isElectronApp && activeTab === SETTINGS_TABS.Keys && (
+          <CSSTransition {...cssTransitionProps}>
+            <ApiKeysTab />
+          </CSSTransition>
+        )}
```

Comment:
can we reduce repeating code lines by making wrapper below, or any other better way

```
const Wrapper = ({ tab, children }) => (
isElectronApp && activeTab === tab && (
          <CSSTransition {...cssTransitionProps}>
            {children}
          </CSSTransition>
        )
)

<Wrapper tab=SETTINGS_TABS.General>
<GeneralTab />
</Wrapper>

```